uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/view/JMXNavigator.java

```diff
@@ -50,35 +50,34 @@ public void setShouldCollapseLocalProcesses(boolean shouldCollapseLocalProcesses
 	}
 
 	/**
-	 * Tries to connect to a specified node under <i>Local Processes</i> in
-	 * <i>JMX Navigator</i> View.
+	 * Tries to connect to a specified node in <i>JMX Navigator</i> View.
 	 * 
 	 * @param name
 	 *            prefix name of the node
 	 * @return the desired node or null if the searched node is not present
 	 */
-	public TreeItem connectTo(String name) {
+	public TreeItem connectTo(String... strings) {
 
-		log.info("Connecting to '" + name + "'.");
+		log.info("Connecting to '" + strings + "'.");
 		open();
 		List<TreeItem> items = new DefaultTree().getItems();
 
 		for (TreeItem item : items) {
-			if (item.getText().equals("Local Processes")) {
+			if (item.getText().equals(strings[0])) {
 				item.expand();
 				items = item.getItems();
 				break;
 			}
 		}
 
 		for (TreeItem item : items) {
-			if (item.getText().contains(name)) {
+			if (item.getText().contains(strings[1])) {
 				item.select();
 				AbstractWait.sleep(TimePeriod.getCustom(2));
```

Comment:
Please avoid using constant timeouts. If you really need constant timeouts, please use some waiting condition afterwards.

Moreover, use already defined constants such TimePeriod.MEDIUM (3 seconds by default).